docs/EN_US/ContainerizedHPCC/ContainerizedMods/ConfigureValues.xml

```diff
@@ -839,8 +839,43 @@
         format:</para>
 
         <programlisting>ds := DATASET('~remote::hpcc2::somescope1::somelfn1', rec);</programlisting>
+      </sect3>
+
+      <sect3>
+        <title>Preferred Storage</title>
+
+        <para>An option available for each type of cluster; hThor, Thor, and
+        Roxie is the <emphasis>preferredReadPlanes</emphasis> option. </para>
+
+        <para>This option is only significant for logical files which reside
+        on multiple storage planes. When specified, the HPCC Systems platform
+        will seek to read files from the preferred plane(s) if a file resides
+        on them. These preferred planes must exist and be defined in
+        <emphasis>storage.planes </emphasis></para>
+
+        <para>The following is an example of a Thor cluster configured with
+        the preferredDataReadPlanes option. </para>
+
+        <programlisting>thor: 
+- name: thor 
+  prefix: thor 
+  numWorkers: 2
+  maxJobs: 4
+  maxGraphs: 3 
+  preferredDataReadPlanes: 
+  - data-copy
+  - indexdata-copy
+</programlisting>
+
+        <para>In the above example, running a query that reads a file that
+        resides on both "data" and "data-copy" (in that order) normally would
+        read the 1st copy on "data". With that
```

Comment:
first should not be abbreviated

Reply:
@JamesDeFabia - not sure what you mean? 

I think the description is correct. If a file lives on A ("data") and B ("data-copy") it would normally read from A, but with preferredDataReadPlanes set per the example, it will cause it to read from B.